core/templates/pages/story-editor-page/editor-tab/story-node-editor.component.ts

```diff
@@ -466,7 +466,7 @@ export class StoryNodeEditorComponent implements OnInit, OnDestroy {
 
   toggleChapterOutline(): void {
     if (this.windowDimensionsService.isWindowNarrow()) {
-      this.chapterOutlineIsShown = !this.chapterOutlineButtonsAreShown;
+      this.chapterOutlineIsShown = !this.chapterOutlineIsShown;
```

Comment:
This doesn't seem stable to me, if the window is narrow and then becomes narrow again then this will just flip its state, right? Can we make it more stable (i.e. do one thing if the window is narrow and another thing if it's not)?